W-16983641-ADDpersistentObjectStore-47-dm (#2972)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

  </autoscaling>
</deploymentSettings>
----

> | `persistentObjectStore` | Configures the Mule application to use a persistent object store. By default, it is set to `false`.

Reviewer: ```suggestion
| `persistentObjectStore` | Configures the Mule application to use a persistent object store. By default, it's set to `false`.
```
